### DISCUSSION TITLE
혹시 Hibernation fixup.kext 의 -hbfx-disable-patch-pci 를 사용 해보셨나요?

### DISCUSSION THREAD

**Discussion by amane1234** [lshbluesky/OC-GA-B250M-DS3H-Hackintosh#42] [Ideas]
저도 HD 630을 igpu로 사용하는 해킨토시 기기가 있는데

hibernationFixup.kext 와 boot-args에 -hbfx-disable-patch-pci 사용을 하니

S4 wake시에 검은화면 나오는 오류가 해결이 되더라고요.

혹시라도 해결이 될수 있을까 싶어서 남겨봅니다.

**Comment by lshbluesky** [comment]
좋은 정보 감사드립니다. :)
나중에, 한 번 적용해 보겠습니다.
